build_resources/linux/after-install.tpl

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+set -eu
+
+###
+# Default after-install.tpl copied from electron-builder.
+# https://github.com/electron-userland/electron-builder/blob/v24.4.0/packages/app-builder-lib/templates/linux/after-install.tpl
+###
+
+# SUID chrome-sandbox for Electron 5+
+chmod 4755 '/opt/${sanitizedProductName}/chrome-sandbox' || true
+
+# update-mime-database and update-desktop-database might be missing from minimal variants of some
+# Linux distributions.
+if hash update-mime-database 2>/dev/null; then
+  update-mime-database /usr/share/mime || true
+fi
+if hash update-desktop-database 2>/dev/null; then
+  update-desktop-database /usr/share/applications || true
+fi
+
+
+###
+# Custom after-install.tpl script.
```

Comment:
Isn't there a way to avoid overwriting the Electron's project `after-install` script, just add an additional script to be run after the default?

If they make any change there in the next releases, we would miss it.

Reply:
I don't agree with it because this custom script wraps up default one doing some polish work over it. Maybe we should have a review process? Idk, maybe in the release process? It could be also a QA step? What do you think?

Reply:
Ok, I hadn't noticed you made changes over the default script from Electron as well.

I guess when we next upgrade Electron would be the time to check this, but I'm not sure where we should document this.